### ISSUE TITLE
Operational?

### ISSUE THREAD

**Issue by NLLAPPS** [xdtianyu/WebDav#1]
Is this operational? I am looking for  an okhttp depended web dav library.
Thanks

**Comment by xdtianyu** [comment]
Yes, this project is still under development. I'm still busy in other project, I will update this later.

**Comment by NLLAPPS** [comment]
Thanks very much, I cannot wait and happy to test it when needed.

**Comment by NLLAPPS** [comment]
I can see the implementation at

https://github.com/xdtianyu/Gallery/blob/master/app/src/main/java/org/xdty/gallery/model/WebDavMedia.java

Are you planing to add PUT and DELETE methods?

**Comment by xdtianyu** [comment]
Yes, I will add more methods to it.
